CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
+### Fixed
+- **BREAKING**: Update `bn.js` from `4.11.6` to `^5.2.1` ([#12](https://github.com/MetaMask/ethjs-unit/pull/12))
```

Comment:
Why is this breaking? What should we be aware of?

Reply:
Oh, I guess `getValueOfUnit` returns a BN, so that has a different interface than it does now. This is good then 👍🏻